server/gateway/src/odspUtils.ts

```diff
@@ -6,15 +6,20 @@
 import { IFluidResolvedUrl } from "@fluidframework/driver-definitions";
 import { OdspDriverUrlResolver } from "@fluidframework/odsp-driver";
 import {
-    getDriveItemByRootFileName,
+    createErrorFromResponse,
+    getAsync,
     getOdspRefreshTokenFn,
     IClientConfig,
+    IOdspAuthRequestInfo,
+    IOdspDriveItem,
     IOdspTokens,
+    putAsync,
 } from "@fluidframework/odsp-utils";
 
 const spoTenants = new Map<string, string>([
     ["spo", "microsoft-my.sharepoint.com"],
     ["spo-df", "microsoft-my.sharepoint-df.com"],
+    ["spo-shared", "microsoft.sharepoint.com"],
```

Comment:
Do we want to support `microsoft.sharepoint-df.com` ?

Reply:
I tried putting it in :( but I kept on getting an error with too many server redirects

Reply:
I think we can probably add it later.